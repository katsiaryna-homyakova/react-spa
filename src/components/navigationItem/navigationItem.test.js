import React from 'react';
import { render } from 'enzyme';
import NavigationItem from './navigationItem';

describe('NavigationItem component', () => {
  it('renders correctly', () => {
    const component = render(<NavigationItem />);
    expect(component).toMatchSnapshot();
  });
});
