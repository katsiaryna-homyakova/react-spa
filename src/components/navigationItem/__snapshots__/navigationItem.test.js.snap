// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationItem component renders correctly 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "href": "/",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "fas fa-search navigation-item",
        },
        "children": Array [],
        "name": "i",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
    ],
    "name": "a",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "href": undefined,
    },
    "x-attribsPrefix": Object {
      "href": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
